src/js/tabs/receiver.js

```diff
@@ -612,6 +646,11 @@ function updateInterpolationView() {
     $('.tab-receiver .rcSmoothing-input-type').show();
     $('.tab-receiver .rcSmoothing-derivative-manual').show();
     $('.tab-receiver .rcSmoothing-input-manual').show();
+    if (semver.gte(CONFIG.apiVersion, "1.42.0")) {
+        if (parseInt(RX_CONFIG.rcSmoothingDerivativeCutoff) === 0 || parseInt(RX_CONFIG.rcSmoothingInputCutoff) === 0) {
```

Comment:
Usually we must store in the MSP objects the elements in the correct form. If this two elements are integers, they must be stored as integers (and not as strings) so the `parseInt` here will not be necessary. If it is I suppose something is wrong in other place, when reading the input field and we store it in the variable.

I see it is used in others `if` later too in the actual code, so if you can, I think is time to fix all of them.

Reply:
It was not necessary on any of the `if` statements below so I just removed it.